examples/pysdl2.py

```diff
@@ -36,33 +43,43 @@
   https://github.com/neilmunday/pes/blob/master/lib/pes/ui.py
 """
 
+import argparse
+import logging
 import sys
+
```

Comment:
PEP 8 requires two blank lines here. There are several PEP8 issues in this commit that should be fixed. There is a tool that can check code for pep8 issues.